app/components/Analyst/Project/Claims/ClaimsForm.tsx

```diff
@@ -290,7 +290,7 @@ const ClaimsForm: React.FC<Props> = ({ application, isExpanded }) => {
         saveBtnText={
           formData?.claimsFile && excelFile ? 'Save & Import' : 'Save'
         }
-        title="Claims"
+        title="Claims & Progress report"
```

Comment:
Looks like the issue was poorly named but it's `Claims & progress reports` according to the acceptance criteria :)

https://github.com/bcgov/CONN-CCBC-portal/issues/2418

Reply:
Great catch, updating :)